bin/run-addon

```diff
@@ -79,6 +81,14 @@ while [[ -n "$1" ]] ; do
   esac
 done
 
+open="open $binary" # assumes Mac OS, availability of 'open' command
+unamestr=$(uname)
+if [[ "$unamestr" == "Linux" ]]; then
+  open=$binary
+# elif [[ "$unamestr" == "" ]]; then
+#  # TODO add cases for Windows, FreeBSD, etc.
```

Comment:
FreeBSD should be the same as Linux, which is maybe a good reason to test for Mac and default to Linux.  On Windows the development process is probably very broken in ways I haven't even considered!